arduino-ide-extension/src/browser/contributions/archive-sketch.ts

```diff
@@ -29,10 +29,12 @@ export class ArchiveSketch extends SketchContribution {
   }
 
   private async archiveSketch(): Promise<void> {
-    const [sketch, config] = await Promise.all([
-      this.sketchServiceClient.currentSketch(),
-      this.configService.getConfiguration(),
-    ]);
+    const config = this.configService.tryGetConfig();
+    if (!config) {
+      return;
```

Comment:
TODO: fallback to any path.

Reply:
https://github.com/arduino/arduino-ide/pull/1774/commits/ae7429a257673879ee1c8b62b9cc79ffce42b6e6

Reply:
https://github.com/arduino/arduino-ide/pull/1774/commits/784c911d54afc22a023cf617ed0c8700dfe6466c